docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -196,6 +196,298 @@ tcld apikey disable --id <api-key-id>
+- **gRPC header**: Connection information supplied by the user in the Temporal CLI or SDK when connecting to Temporal Cloud using an API Key. Currently, in most SDKs and the CLI, the gPRC header should include a Bearer Token with the API Key and a Namespace header.
+
+## API Keys for Client Connections
+
+Temporal Cloud supports secure programmatic access through API Key Authentication.
+
+#### Overview
+
+Temporal Cloud API Keys facilitate identity-level authentication, for Users and Service Accounts. Each key is unique to a single identity and ties directly to the identity's Role-Based Access Control (RBAC) permissions, ensuring secure and appropriate access.
+
+The process is as follows:
+
+API Key (authentication) → User or Service Account (identity) → RBAC (authorization)
+
+These API Keys are used to authenticate Temporal Cloud operator tools, including Temporal Cloud CLI (tcld) and the Temporal Cloud Operations API (Cloud Ops API), and Client Connection tools, including the Temporal CLI and SDKs.
+
+
+#### API Key Summary
+
+Temporal Cloud API Keys functionality is designed for 3 types of use:
+
+1. User created API Keys - if enabled, users can create, delete, and update their own API Keys through the Cloud UI or tcld.
+2. Service Account created API Keys - Global Administrators can create, delete, and update API Keys for Service Accounts. 
+3. API Key governance - Temporal Cloud provides affordances for Global Administrators to govern and control API Key use within their accounts. Global Administrators can enable/disable API Keys for their accounts, and disable/delete API Keys for any User and/or Service Account.
+
+By default, API Keys must be enabled by a Global Administrator before use.
+
+#### Get started
+
+Prerequisites
+
+- Access to the Temporal Cloud UI or Temporal Cloud CLI (tcld) - to create an API Key
+- Enable access to API Keys for your Account
+- Create a new Namespace that allows API Key connectivity.
+
+#### Temporal CLI
+
+To use your API Key with the Temporal CLI, you can pass in the API Key to each command using the `--api-key` flag or an environmental variable (best practice).
+
+In addition to passing in the API Key, the following client options are required:
+
+- `--address`: Provide the Namespace's gRPC endpoint located in the Namespace UI in the gRPC endpoint box.
+  - For API Key connectivity, the endpoint format is `<region>.<cloud provider>.api.temporal.io:7233`
+  - Can be set using an environmental variable
+- `--namespace`: Provide the namespace.accountId that appears at the top of the Namespace page in the UI.
+  - Format is `<namespace.accountId>`
+  - Can be set using an environmental variable
+- `--grpc-meta "temporal-namespace="`: Provide the namespace.accountId again as grpc-metadata.
+- `--tls` for a secure connection with the appropriate options
+  - Potentially, with `--tls-disable-host-verification` to disable host verification
```

Comment:
hmm, this seems slightly sketchy. will need SDK team to confirm.

Reply:
+1 on this, is it required to disable host verification in production? If it is, what do we have to do to make it work with host verification?

Reply:
I made a ticket for the SDK team to double-check everything and also sent it to product (aka Jonathan) for feedback. Since we need to publish it today, I'll go ahead and do that now. We can always make edits later if needed.

Reply:
No, you shouldn't need to do this typically, but looks like it's already not on the docs site :+1: